linebot/models/filter.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may
+#  not use this file except in compliance with the License. You may obtain
+#  a copy of the License at
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+
+"""linebot.models.filter module."""
+
+from __future__ import unicode_literals
+
+from abc import ABCMeta
+
+from future.utils import with_metaclass
+
+from .base import Base
+
+
+class Filter(with_metaclass(ABCMeta, Base)):
+    """Filter.
```

Comment:
`DemographicFilter` corresponds to the key `filter.demographic`.
As other keys `filter.*` will be added, I think we should define `Filter` like

```py
filter=Filter(demographic= DemographicFilter(...))
```

Reply:
Done, I've make DemographicFilter as a base class of demographic filters.

Reply:
Thanks.